semaphore/build_push.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e -x -u
+
+# bash script should be called with AWS environment (dev / dev-green / dev-blue / prod)
+# Other required configuration:
+# * APP
+# * DOCKER_REPO
+awsenv=$1
+
+# build docker image and tag it with git hash and AWS environment
+githash=$(git rev-parse --short HEAD)
+docker build -t $APP:latest .
```

Comment:
This also needs to include the Docker login changes: for an example, see https://github.com/mbta/tablespoon/commit/3ab2537550107eb64d563048875d2d8d960c85e4